contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -2,62 +2,155 @@ category: application
 doc: |
   This is the most general application definition for multidimensional
   photoelectron spectroscopy.
+  
+  Groups and fields are named according to the
+  `ISO 18115-1:2023`_ specification as well as the `IUPAC Recommendations 2020`_.
+  
+  .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
+  .. _IUPAC Recommendations 2020: https://doi.org/10.1515/pac-2019-0404
+symbols:
+  doc: |
+    The symbols used in the schema to specify e.g. dimensions of arrays
+  n_transmission_function: |
+    Number of data points in the transmission function.
 type: group
 NXmpes(NXobject):
   (NXentry):
+    definition:
+      \@version:
```

Comment:
This is inconsistent with at least NXem and NXapm, NXmx, attribute version is a first-order child of NXentry instead of a first-order child of definition

Reply:
I would argue that it is the definition we are versioning? Then again, it has the same effect, so we should agree on something. Maybe we can discuss in the TF.

Reply:
According to the nexus manual this is the correct way to do it: https://manual.nexusformat.org/classes/base_classes/NXentry.html#nxentry

Reply:
@domna is right, learned this now, indeed NXmx is doing it inconsistently https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx